### DISCUSSION TITLE
How to use in CMD env?

### DISCUSSION THREAD

**Discussion by mrchris2000** [JanDeDobbeleer/oh-my-posh#1045] [Troubleshoot] [answered]
Can oh-my-posh be used by default in a cmd environment?
Currently it runs an exits which would imply it should work, but unlike powershell the install instructions don't detail how to update the cmd profile(?) effectively.

![image](https://user-images.githubusercontent.com/5471394/137196427-ea26a5f5-f0e1-469f-a90b-219b52116835.png)

The instructions talk about the theme file but not what needs to be done to get the app to start.
I considered adding this as an issue, but the current one seems to just refer back to the theme locations.

**Comment by JanDeDobbeleer** [comment, marked as answer]
You can use clink to set this up but is not officially supported.

**Reply by mrchris2000** [reply]
That works a charm, thank you.
